CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 5.4
```

Comment:
See, we have a little problem here, in that we're manually referencing version numbers that don't exist yet. What if we merged another PR before this? We really need to automate this. I think `pr-bumper` can prepend some text to `CHANGELOG.md` from the description in the PR.